Tools/TestWebKitAPI/cocoa/TestDownloadDelegate.mm

```diff
@@ -59,10 +59,10 @@ - (void)download:(WKDownload *)download didReceiveAuthenticationChallenge:(NSURL
 }
 
 #if HAVE(MODERN_DOWNLOADPROGRESS)
-- (void)_download:(WKDownload *)download decidePlaceholderPolicy:(void (^)(_WKPlaceholderPolicy))completionHandler
+- (void)_download:(WKDownload *)download decidePlaceholderPolicy:(void (^)(_WKPlaceholderPolicy, NSURL *))completionHandler
 {
     _callbackRecord.append(DownloadCallback::DecidePlaceholderPolicy);
-    completionHandler(_WKPlaceholderPolicyDisable);
+    completionHandler(_WKPlaceholderPolicyDisable, nil);
```

Comment:
The new delegate callbacks need to be tested.  We should also have a test with a non-null URL here and if possible we should verify that something happens with that URL.